src/cljs/main/broadfcui/page/test_login.cljs

```diff
@@ -0,0 +1,50 @@
+(ns broadfcui.page.test-login
+  (:require
+   [dmohs.react :as react]
+   [broadfcui.components.buttons :as buttons]
+   [broadfcui.components.foundation-tooltip :as tooltip]
+   [broadfcui.common.icons :as icons]
+   [broadfcui.common.input :as input]
+   [broadfcui.common.style :as style]
+   [broadfcui.nav :as nav]
+   [broadfcui.utils :as utils]
+   [broadfcui.utils.user :as user]
+   ))
+
+(react/defc Page
+  {:render
+   (fn [{:keys [this refs]}]
+    [:div {}
+      [:div {}
+        "Token:"
+        [input/TextField {:ref "force-token"
+                          :autoFocus true
+                          :type "password"
```

Comment:
Is there a reason this needs to be a password field? The tests run headless, I believe, so this seems to be adding unnecessary complication with the changes required to support this.

Reply:
an  overabundance of caution. A developer is testing their automation scripts, cuts and pastes their token into a browser, gets distracted, walks over to the bathroom ... if having this be a password field causes automation problems, we can switch it back.